src/Utilities/TrackedDependencies/CanonicalTrackedOutputFiles.cs

```diff
@@ -32,12 +32,7 @@ public class CanonicalTrackedOutputFiles
         #region Properties
 
         // Provide external access to the dependencyTable
-#if WHIDBEY_VISIBILITY
-        internal
-#else
-        public
-#endif
-        Dictionary<string, Dictionary<string, DateTime>> DependencyTable
+        internal Dictionary<string, Dictionary<string, DateTime>> DependencyTable
```

Comment:
I took a chance here, the internal seemed sufficient. I may be wrong, but I tried to minimize visibility.